charmed-bind/snap/snapcraft.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: charmed-bind
```

Comment:
Does the name remain the same?

Reply:
I thought about renaming it bind-app in the process, to be more aligned with dns-policy-app. Would you agree with that ?

Reply:
Sure